.github/workflows/npm-publish.yml

```diff
@@ -116,6 +116,39 @@ jobs:
         path: |
           node/dist/vectordb-linux*.tgz
 
+  node-windows:
+    runs-on: windows-2022
+    # Only runs on tags that matches the make-release action
+    if: startsWith(github.ref, 'refs/tags/v')
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [x86_64-pc-windows-msvc]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Protoc v21.12
+        working-directory: C:\
+        run: |
+          New-Item -Path 'C:\protoc' -ItemType Directory
+          Set-Location C:\protoc
+          Invoke-WebRequest https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protoc-21.12-win64.zip -OutFile C:\protoc\protoc.zip
+          7z x protoc.zip
+          Add-Content $env:GITHUB_PATH "C:\protoc\bin"
```

Comment:
Powershell, impressive!

(FYI GitHub action machines [come with git bash installed](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#shells), so you can use bash instead if you want in the future).